data/avro-schema/src/main/resources/avro/net/corda/data/persistence/DeleteEntityByPrimaryKey.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "DeleteEntityByPrimaryKey",
+  "doc": "Delete an entity by its id.  See also {@link EntityRequest#request}",
```

Comment:
I think `DeleteEntityById` was actually correct. It's the JPA object `@Id` annotation that is relevant, it doesn't actually _have_ to be a primary key in the Db.

Reply:
So you want it changed back to DeleteById then?

Reply:
I think that would make more sense in terms of an API.
Sorry, I should have been clearer in my original comment.